server/sandbox.go

```diff
@@ -30,8 +31,15 @@ type sandbox struct {
 	metadata       *pb.PodSandboxMetadata
 }
 
+type sysctl struct {
+	Name  string
```

Comment:
I believe you can unexport `Name` and `Value` by using lowercase first letters (since we use this type just in this package)